docs/project/release-process.md

```diff
@@ -28,3 +28,30 @@ For Feast maintainers, these are the concrete steps for making a new release.
 4. Try the dry run first with your personal access token. If this succeeds, uncheck `Dry Run` and run the release workflow.
 5. All of the jobs should succeed besides the UI job which needs to be released separately. Ping a maintainer on Slack to run the UI release manually.
 6. Try to install the feast release in your local environment and test out the `feast init` -> `feast apply` workflow to verify as a sanity check that the release worked correctly.
+
+### (for minor releases) Post-release steps
+1. Create a new branch based on master (i.e. v0.22-branch) and push to the main Feast repo. This will be where cherry-picks go for future patch releases and where documentation will point.
+2. Write a summary of the release in the GitHub release 
+   1. By default, Semantic Release will pull in messages from commits (features vs fixes, etc). But this is hard to digest still, so it helps to have a high level overview.
+
+### Update documentation
```

Comment:
Are there any credentials or things that maintainers need to set up before this is possible?

Reply:
Added a note to contact me in Slack